presto-main/src/main/java/com/facebook/presto/server/remotetask/TaskInfoFetcher.java

```diff
@@ -190,7 +190,7 @@ private synchronized void sendNextRequest()
         }
 
         Request request = prepareGet()
-                .setUri(uriBuilderFrom(taskStatus.getSelf()).addParameter("summarize").build())
```

Comment:
I think we should only do this for `EXPLAIN ANALYZE` and only for the "final" fetch